.setup/data/numeric_ids.txt

```diff
@@ -0,0 +1,35 @@
+074746041
```

Comment:
where is this file used and do we need it?

Reply:
these are the 35 ids from the pdf of handwritten RINS